frontend/src/components/Comments/CommentsSection.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+type CommentsSectionProps = {
+  restaurantId: string;
+};
+
+function CommentsSection({ restaurantId }: CommentsSectionProps) {
+  const [comments, setComments] = useState<{ text: string }[]>([]);
```

Comment:
Please use object types and not an array of a string. What if you want to show the date and author of the comment?
Also create a custom hook instead of useState.